docs/advanced/experimental/indexers.md

```diff
@@ -60,6 +60,7 @@ For a showcase code, check our [integration tests](https://github.com/jina-ai/ex
 
 The split between indexing and search Flows allows you to continuously serve requests in your application (in the search Flow), while still being able to write or modify the underlying data. Then when you want to update the state of the searchable data for your users, you perform a dump and rolling update.
 
+(dump-rolling-restart)=
 ## Dump and rolling update
```

Comment:
```suggestion
## Dump and Rolling Update
```

I think we use [Title case](https://en.wikipedia.org/wiki/Title_case) for headings